pkg/strategy/common/sync.go

```diff
@@ -26,6 +27,10 @@ func SyncActiveOrder(ctx context.Context, ex types.Exchange, orderQueryService t
 		return isOrderUpdated, err
 	}
 
+	if updatedOrder == nil {
+		return isOrderUpdated, fmt.Errorf("updatedOrder is nil and there is no error, please check it")
```

Comment:
use `unexpected error, order object is a nil pointer, please check common.SyncActiveOrder()`